### PR TITLE
Remove padding from the KID value in backendJWT

### PR DESCRIPTION
**Pull request by jhivandebenoit**
### Purpose
Increment apim common gatway dependency.
Removed some redundant code
Removed padding from kid property value

### Automation tests
 - Unit tests added: No
 - Integration tests added: No
 
### Tested environments

Not Tested

---
#### Maintainers: Check before merge
- [x] Assigned 'Type' label
- [x] Assigned the project
- [x] Validated respective github issues
- [x] Assigned milestone to the github issue(s)

### CODE REVIEW

File: enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/config/ConfigHolder.java [hunk 1]
```diff
@@ -493,7 +492,7 @@ private void populateBackendJWKSConfiguration(JWTGenerator jwtGenerator) {
                 RSAKey jwk = new RSAKey.Builder(publicKey)
                         .keyUse(KeyUse.SIGNATURE)
                         .algorithm(getJWKSAlgorithm(jwtGenerator.getSigningAlgorithm()))
-                        .keyID(getJWKSThumbprint(cert))
+                        .keyID(generateThumbprint("SHA-256", cert, false))
```
Comment (VirajSalaka) [inline enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/config/ConfigHolder.java:495]: Let's import JWTUtil and invoke like JWTUtil.generateThumbprint

Comment (jhivandebenoit) [reply enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/config/ConfigHolder.java]: ack

**Review by VirajSalaka — APPROVED** [review]

**Comment by choreo-cicd** [comment]
[failed] Controlplane cluster : dev-deployment-v2 : 20230103.3

**Comment by choreo-cicd** [comment]
[failed] DataplaneE1 cluster : dev-deployment-v2 : 20230103.3

**Comment by choreo-cicd** [comment]
[failed]  : dev-deployment-v2 : 20230103.3

**Comment by choreo-cicd** [comment]
[failed] DataplaneE2 cluster : dev-deployment-v2 : 20230103.3

**Comment by choreo-cicd** [comment]
[succeeded]  : dev-deployment-v2 : 20230103.5

**Comment by choreo-cicd** [comment]
[succeeded] Controlplane cluster : dev-deployment-v2 : 20230103.5

**Comment by choreo-cicd** [comment]
[succeeded] DataplaneE2 cluster : dev-deployment-v2 : 20230103.5

**Comment by choreo-cicd** [comment]
[succeeded] DataplaneE1 cluster : dev-deployment-v2 : 20230103.5

**Comment by choreo-cicd** [comment]
[succeeded] DataplaneE1 cluster : stage-deployment-v2 : 20230104.3

**Comment by choreo-cicd** [comment]
[succeeded] Controlplane cluster : stage-deployment-v2 : 20230104.3

**Comment by choreo-cicd** [comment]
[succeeded] Controlplane cluster : prod-deployment-v2 : 20230109.2

**Comment by choreo-cicd** [comment]
[succeeded] DataplaneE1 cluster : prod-deployment-v2 : 20230109.2

**Pull request merged**